packages/vscode-apollo/package.json

```diff
@@ -100,6 +100,16 @@
         "embeddedLanguages": {
           "meta.embedded.block.graphql": "graphql"
         }
+      },
+      {
+        "injectTo": [
+          "source.dart"
+        ],
+        "scopeName": "inline.dart.python",
```

Comment:
It looks like this is wrong. You'd need it to be `inline.graphql.dart`. That fixed the highlighting issue for me :)

Reply:
🤦‍♂️thanks for finding that! I ran the extension in debug mode with this change and everything worked.